C-Plus-Plus/cryptography/VernamCipher.cpp

```diff
@@ -0,0 +1,67 @@
+//Vernam Cipher
+
+// Vernam Cipher is a symmetric key cryptographic algorithm and is one of the transposition technique that converts plain text into ciphertext.
+// In this algorithm the size of the key and plain text must be same.
+
+
+#include<bits/stdc++.h>
+using namespace std;
+
+string encrypt(string plain,string key){
+	string enc="";
+	int n=key.length();
+  for(int i=0;i<n;i++){
```

Comment:
Use a consistent indentation

Reply:
I've used the consistent indentation but on pushing the code the indentation is getting change on its own.  Rest all the changes I've made.